src/main/play-doc/general/MigrationGuide.md

```diff
@@ -33,18 +33,325 @@ In additional to disabling Elasticsearch via configuration, a service lookup mus
   -Dcontrail.syslog.server.service-locator.enabled=off
 ```
 
-### Service ports
+## Service endpoint declaration
 
-**The services is now deprecated in lieu of [request acl](RequestAclConfiguration).**
+The services endpoint declaration defines the protocol, port, and/or path under which your service will be addressed to the outside world.
 
-The services define the protocol, port, and/or path under which your service will be addressed to the outside world on. For example, if http and port 80 are to be used to provide your services and then the following expression can be used to resolve `/myservice` on:
+**From 1.2 onwards, the service ports are deprecated in lieu of [[request ACL|AclConfiguration]].**
 
-```json
-    endpoints        = {
-      "angular-seed-play" = {
-        protocol  = "http"
-        bind-port = 0
-        services  = ["http:/myservice"]
+The [[request ACL|AclConfiguration]] in conjunction with [[dynamic proxy configuration|DynamicProxyConfiguration]] provides additional proxy mapping flexibility on top of what was previously provided by the service ports. This will allow operations to have a complete control on the proxy configuration.
+
+### Migrating HTTP-based services
+
+Some HTTP based service definition can be migrated by supplying its [[request ACL|AclConfiguration]] equivalent, while others require a customized HAProxy configuration.
+
+#### Migrating services definition
+
+The following HTTP based service definition can be migrated by supplying its [[request ACL|AclConfiguration]] equivalent.
+
+##### Services mapped to root path
+
+Given the following example where the service URI was "http://my-service", ConductR will interpret the first path component (`my-service`) as the service name for the purposes of service lookup. By default ConductR will then remove the first path component when rewriting the request. This means that your application or service will be deployed as root context or `/`.
```

Comment:
@markusjura and @edwardcallahan : `my-service` is now used as the value of the service name